angel_system/global_step_prediction/global_step_predictor.py

```diff
@@ -144,6 +144,7 @@ def get_activity_order_from_config(self, config_fn):
     def compute_average_TP_activations(
         self,
         coco_train: kwcoco.CocoDataset,
```

Comment:
Do we want to maybe call this "_predictions" instead of "_train"? Maybe its just me that finds this naming confusing.

Reply:
Honestly I do like `coco_train`, so I can keep straight in my head that for the GSP, and in the little universe that is `compute_average_TP_activations`, this is the data I'm "training" on. (It doesn't matter much to me that this data has provenance in which it was something else.)

However, I don't find `coco_preds` to be terribly confusing either! So I just switched to `coco_preds`. I'd rather more of us feel pretty good about the naming.

Reply:
And maybe it should matter more to me that this data has recent provenance wherein it was something else.

I will ponder this